lib/utils/unsupported.js

```diff
@@ -7,7 +7,8 @@ var supportedNode = [
   {ver: '10', min: '10.0.0'},
   {ver: '11', min: '11.0.0'},
   {ver: '12', min: '12.0.0'},
-  {ver: '13', min: '13.0.0'}
+  {ver: '13', min: '13.0.0'},
+  {ver: '14', min: '14.0.0'}
```

Comment:
thank you for doing this in advance :-)

could perhaps this logic be changed to use semver so that every new node version is always treated as supported unless explicitly marked as not?

Reply:
yeah, it's in our radar to change that for v7 👍

Reply:
it’d also be nice tho for the last 6.x build to not report that new nodes are unsupported forever :-)

Reply:
right, that's a good point! I agree 👍 that could be a good community contribution _wink wink_

Reply:
Filed https://github.com/npm/cli/pull/697 (i'll need some help)